### PR TITLE
Accessibility label in Layout

### PR DESCRIPTION
**Pull request by josepmartins**
- [x] Add accessibility reviewed Label in the Layout
- [x]  Adjust the status label styles
- [x] Upgrade `Figma` custom icon to match Octicons guidelines
- [x] Use `Book` icon for Lookbook & Storybook links
- [x] Bump Octicons to use `AccessibilityInsetIcon`
- [x] Responsive `flex-wrap` behavior.
- [x] Add `Rails` + `React` links to cover design guidelines needs

👁️ Change `Caption` frontmatter for changes: https://primer-3c379c0b2a-26493675.drafts.github.io/components/caption

Design Guidelines - Reviewed
Design Guidelines - Not reviewed
Component Page - Reviewed
Mobile

### CODE REVIEW

**Comment by josepmartins** [comment]
Temporary keeping the `Caption` frontmatter variables for testing reasons

File: docs/content/components/caption.mdx [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 title: Caption
-status: New
+status: Stable
 source: https://github.com/primer/doctocat/blob/master/theme/src/components/caption.js
+figma: https://github.com/primer/doctocat/blob/master/theme/src/components/caption.js
+storybook: https://github.com/primer/doctocat/blob/master/theme/src/components/caption.js
 componentId: caption
+description: An ActionList is a list of items that can be activated or selected. ActionList is the base component for many menu-type components, including ActionMenu and SelectPanel.
+a11yReviewed: true
 ---
```
Comment (josepmartins) [inline docs/content/components/caption.mdx:10]: Keeping this variables temporary for testing reasons

File: docs/content/components/caption.mdx [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
 title: Caption
-status: New
+status: Stable
 source: https://github.com/primer/doctocat/blob/master/theme/src/components/caption.js
+figma: https://github.com/primer/doctocat/blob/master/theme/src/components/caption.js
+storybook: https://github.com/primer/doctocat/blob/master/theme/src/components/caption.js
 componentId: caption
+description: An ActionList is a list of items that can be activated or selected. ActionList is the base component for many menu-type components, including ActionMenu and SelectPanel.
```
Comment (josepmartins) [inline docs/content/components/caption.mdx:8]: Keeping this variables temporary for testing

**Review by colebemis — APPROVED** [review]
Love the new design 💖

**Pull request merged**